src/tests/functional/shared_test_classes/include/shared_test_classes/base/benchmark.hpp

```diff
@@ -0,0 +1,181 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cctype>
+#include <iostream>
+
+#include "layer_test_utils.hpp"
+#include "ov_subgraph.hpp"
+
+namespace LayerTestsDefinitions {
+
+template <typename BaseLayerTest>
+class BenchmarkLayerTest : public BaseLayerTest {
+    static_assert(std::is_base_of<LayerTestsUtils::LayerTestsCommon, BaseLayerTest>::value,
+                  "BaseLayerTest should inherit from LayerTestsUtils::LayerTestsCommon");
+
+public:
+    void Run(const std::initializer_list<std::string>& names,
```

Comment:
quite confusing with LayerTestsUtils::LayerTestsCommon::Run() honestly
may be RunBenchmarking

Reply:
I okay with reusing name `Run` because end-user should not care what exactly single layer test is testing.
I think that `Run` is made a virtual method to abstract what does it mean for particular test to "Run".

As for me it would make sense to rename it to another method if I added this method in `SubgraphBaseTest` hierarchy.

But if you'll insist on renaming it to `RunBenchmarking`, I could do it

Reply:
Oh, I've just figured out that this method is not virtual ...
Okay, than it make sense to rename it to `RunBenchmark`

Reply:
Done